api/src/main/java/org/pmiops/workbench/api/CloudTaskEnvironmentsController.java

```diff
@@ -43,7 +43,6 @@ public ResponseEntity<Void> deleteUnsharedWorkspaceEnvironmentsBatch(
         String.format(
             "Deleting unshared environments for %d workspaces.", workspaceNamespaces.size()));
 
-    // temp timing for batch sizing
```

Comment:
I like it, so I'm keeping it.  no longer "temp".  Expanded to all of CloudTaskUserController.  May expand elsewhere later.